specification/datafactory/resource-manager/readme.python.md

```diff
@@ -0,0 +1,26 @@
+## Python
+
```

Comment:
I remember they have python configuration in readme.md. Maybe we should move it out here.

Reply:
this is just for demo purpose